### PR TITLE
Beoble 1396

### PR DESCRIPTION
**Pull request by bamnenim-kryptonium**

### CODE REVIEW

File: packages/beoble/components/DesktopView/Button/Button.tsx [hunk 1]
```diff
@@ -108,8 +108,8 @@ const buttonMeta: ButtonMeta = {
     backgroundColor: 'white',
   },
   LOGIN: {
-    icon: 'MetaMask',
-    text: 'Connect Wallet',
+    icon: 'BeobleWhite',
+    text: 'LogIn to beoble',
```
Comment (vanilla-james) [inline packages/beoble/components/DesktopView/Button/Button.tsx:112]: text should be Login (one word without PascalCase?)

File: packages/beoble/components/DesktopView/PanelPages/ConnectPagePanel.tsx [hunk 2]
```diff
@@ -115,11 +86,16 @@ export function Index() {
...
         <Header>Start chatting with beoble.</Header>
         <ButtonsContainer>
-          <Button
-            buttonType="LOGIN"
-            onClick={() => handleClick('MetaMask')}
-            isSelectedOverride={connectingState.isConnecting}
-          />
+          <ConnectButton />
+          {isConnected && (
+            <div style={{ marginTop: 20 }}>
```
Comment (vanilla-james) [inline packages/beoble/components/DesktopView/PanelPages/ConnectPagePanel.tsx:91]: separate out as StyledComponent?

File: packages/beoble/components/Popup/index.tsx [hunk 3]
```diff
@@ -1,9 +1,9 @@
 import { PulseLoader } from 'react-spinners';
 import styled from 'styled-components';
+import { useWeb3Atom } from '@beoble/react';
 import { utils } from '@beoble/js-sdk';
 import { FlexCenter } from '../../../react/src/styles/commons';
-import { useAtomValue } from 'jotai';
-import { ethAccountAtom } from '../../../react/src/atoms/useEthereumAtom';
+import MetaMaskSvg from 'packages/beoble/styles/svgs/MetaMaskSvg';
```
Comment (vanilla-james) [inline packages/beoble/components/Popup/index.tsx:6]: use relative import for importing files from same package?

wrongly using absolute imports in the past has caused ci/cd build errors and other weird runtime errors

Comment (vanilla-james) [inline packages/beoble/components/Popup/index.tsx:6]: I made an `Icon` component that you can pass the name as a props to declare what icon you want to use
> packages/beoble/components/DesktopView/Icon/Icon.tsx

example usage)
> packages/beoble/components/Popup/index.tsx: line 28 
`
<Icon icon="MetaMask" size="lg" />
`

File: packages/react/src/hooks/queries/user/useGetUserChatRoomSetting.ts [hunk 4]
```diff
@@ -7,6 +7,7 @@ export default function useGetUserChatRoomSetting(user_id: string) {
   const { Beoble } = useBeoble();
 
   const getUserChatRoomSetting = async () => {
+    if (!Beoble.auth.isAuthorized) return [];
```
Comment (vanilla-james) [inline packages/react/src/hooks/queries/user/useGetUserChatRoomSetting.ts:10]: see comment above

File: packages/beoble/components/Popup/index.tsx (hunk 3 above)
Comment (bamnenim-kryptonium) [reply packages/beoble/components/Popup/index.tsx:6]: changed to use it

File: packages/react/src/hooks/queries/chatroom/useGetChatroomsByUserId.ts [hunk 5]
```diff
@@ -8,6 +8,7 @@ export default function useGetChatroomsByUserId(userId: string) {
   const { Beoble } = useBeoble();
 
   const getChatroomsByUserId = async (userId: string) => {
+    if (!Beoble.auth.isAuthorized) return [];
```
[the remark this answers is not in the document]
Comment (bamnenim-kryptonium) [reply packages/react/src/hooks/queries/chatroom/useGetChatroomsByUserId.ts:11]: Yeap, found using enabled and returning Promise.reject in the queryFn is the best practice, and applied it

**Pull request merged**